apps/framework-cli/src/framework/transform.ts

```diff
@@ -73,6 +73,17 @@ const getFlows = (): Map<string, Map<string, string>> => {
   return output;
 };
 
+const jsonDateReviver = (key: string, value: unknown): unknown => {
+  const iso8601Format =
```

Comment:
Got this from https://stackoverflow.com/a/14322189 :pray:

Reply:
😱

Reply:
Yea haha, we should revisit further down the line.